api/v1beta1/spannerautoscaleschedule_types.go

```diff
@@ -39,6 +39,8 @@ type SpannerAutoscaleScheduleStatus struct{}
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Cron",type="string",JSONPath=".spec.schedule.cron"
+//+kubebuilder:printcolumn:name="Duration",type="string",JSONPath=".spec.schedule.duration"
 
```

Comment:
```suggestion
//+kubebuilder:printcolumn:name="Additional PU",type="integer",JSONPath=".spec.additionalProcessingUnits"
```

It seems better to add `Additional Processing Units`.

Reply:
```console
$ kubectl get spannerautoscaleschedule -n test
NAME                              CRON           DURATION   ADDITIONAL PU
another-sample                    */25 * * * *   15m30s     300
spannerautoscaleschedule-sample   0 */3 * * *    2h10m      3000
```